src/plugins/html-serializable-plugin.tsx

```diff
@@ -1,7 +1,8 @@
-import { Plugin } from "slate-react";
-import { Rule } from "slate-html-serializer";
+//import { Plugin } from "slate-react";
```

Comment:
This isn't currently used anywhere... But maybe it should be re-incorporated with an updated api?  In my draft, https://github.com/concord-consortium/slate-editor/pull/75/files, CLUE is using an "any" type for the variable Plugin :(